bugbug/bugzilla.py

```diff
@@ -92,10 +100,23 @@ def get_bugs(include_invalid: Optional[bool] = False) -> Iterator[BugDict]:
 
 
 def set_token(token):
+    """Set the Bugzilla API token to be used for authentication.
+
+    Args:
+        token: A string representing the API token.
```

Comment:
You could add a type hint to the token parameter (i.e., `token: str`) and drop the "Args" section here; it is very obvious. 

What do you think?

Reply:
I think that's alright